packages/mdc-textfield/README.md

```diff
@@ -279,6 +279,19 @@ text, validation, and dense UI.
 </div>
 ```
 
+### Sass Mixins
+
+To customize a Text Field's border radii, you can use the following mixins.
+
+#### `mdc-text-field-corner-radius($position, $radius)`
```

Comment:
Does this mixin only actually work (Aka show a rounded corner) if you're using `mdc-textfield--box` or `mdc-textfield--textarea`?

Reply:
Sure does